packages/next/next-server/server/next-server.ts

```diff
@@ -532,7 +532,7 @@ export default class Server {
           }
         }
 
-        if (params && params.path && params.path[0] === 'api') {
+        if (params?.path && params.path[0] === 'api') {
```

Comment:
Can this be further simplified?
```suggestion
        if (params?.path?[0] === 'api') {
```